src/index.html

```diff
@@ -13,6 +13,78 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <section aria-label="Star holder">
+      <section class="stars stars--0">
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+      </section>
+
+      <section class="stars stars--1">
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+      </section>
+
+      <section class="stars stars--2">
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+
+        <div class="stars__star"></div>
+      </section>
```

Comment:
Same comment as above. There should be no empty lines between parent and child elements.